sandbox/src/components/MessageFeedDemo/MessageFeed.js

```diff
@@ -0,0 +1,628 @@
+import React, { useEffect, useState } from "react";
+import ContentSecurityPolicy from "../ContentSecurityPolicy";
+import "./MessageFeed.css";
+
+export default function MessageFeed() {
+  const [messageFeedItems, setMessageFeedItems] = useState([]);
+
+  const prettyDate = value => {
+    let output = "";
+
+    if (typeof value !== "undefined") {
+      const now = new Date().getTime();
+      const seconds = Math.floor(now / 1000);
+      const oldTimestamp = Math.floor(value / 1000);
+      const difference = seconds - oldTimestamp;
+
+      if (difference < 60) {
+        output = `${difference} second(s) ago`;
+      } else if (difference < 3600) {
+        output = `${Math.floor(difference / 60)} min ago`;
+      } else if (difference < 86400) {
+        output = `${Math.floor(difference / 3600)} hour(s) ago`;
+      } else if (difference < 2620800) {
+        output = `${Math.floor(difference / 86400)} day(s) ago`;
+      } else if (difference < 31449600) {
+        output = `${Math.floor(difference / 2620800)} month(s) ago`;
+      } else {
+        output = `${Math.floor(difference / 31449600)} year(s) ago`;
+      }
+    }
+
+    return output;
+  };
+
+  useEffect(() => {
+    /* eslint-disable */
+    const mockResponse = {
```

Comment:
Let's limit items in the mockResponse to only ones that have `https://ns.adobe.com/personalization/message/feed-item` consequences.  That will make it easier to follow.  No need for any other irrelevant rulesets to be included in this sample page.